scaleapi/api.py

```diff
@@ -109,9 +109,28 @@ def _api_request(
         json = None
         if res.status_code == 200:
             json = res.json()
+        elif res.status_code == 409:
+            retry_history = res.raw.retries.history
+            # Example RequestHistory tuple
+            # RequestHistory(method='POST',
+            #   url='/v1/task/imageannotation',
+            #   error=None,
+            #   status=409,
+            #   redirect_location=None)
+            if retry_history != ():
+                # See if the first retry was a 500 error
+                if retry_history[0][3] == 500:
```

Comment:
Just to confirm (since I'm not super sure) but is `500` the only error returned from our API for those race conditions? (and not other 5xx errors).

Reply:
Yeah, the exact error is task creation taking longer than our timeout, so Scale sends a 500 back in the response